pom.xml

```diff
@@ -119,7 +119,7 @@
 					<argLine>-javaagent:${project.build.directory}/agent/zebrunner-agent.jar</argLine>
 					<useSystemClassLoader>false</useSystemClassLoader>
 					<suiteXmlFiles>
-						<suiteXmlFile>${project.build.directory}/test-classes/testng_suites/${suite}.xml</suiteXmlFile>
+						<suiteXmlFile>${project.build.directory}/test-classes/testng_suites/webLogin.xml</suiteXmlFile>
```

Comment:
we should not modify pom.xml to run tests suite
to run some suite you can use this command `mvn test clean -Dsuite=suiteName`